amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -74,11 +94,11 @@ describe('SearchPage', () => {
       props = setupResult.props;
```

Comment:
We should be able to remove the line above:
`window.history.pushState({}, '', '/search?searchTerm=testName&selectedTab=table&pageIndex=1');`